python/cudf/cudf/tests/test_numba_import.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+import subprocess
+import sys
+
+import pytest
+
+skip = True
+try:
+    from ptxcompiler.patch import NO_DRIVER, safe_get_versions
+
+    versions = safe_get_versions()
+    if versions != NO_DRIVER:
+        driver_version, runtime_version = versions
+        if driver_version < (12, 0) and runtime_version > driver_version:
```

Comment:
Do we need the `runtime_version > driver_version` check, or can we test this in more scenarios by unconditionally enabling MVC and testing as long as we have a CUDA 11 runtime version? (and would it be worth doing so, or are we happy this just getting testing in an actual MVC scenario in CI?)

Reply:
Fixed in c81e967.